grunt/config.js

```diff
@@ -31,12 +28,20 @@ module.exports = {
       'src/js/**/*.js',
       'src/modules/**/*.js'
     ],
-    config: 'src/js/config.js',
-    dest: 'dist/<%= pkg.version %>/main.min.js',
+    dev: {
+      entry: 'src/js/main.js',
+      dest: 'src/dist/index.js'
+    },
+    build: {
+      entry: 'src/js/main.js',
+      dest: 'dist/<%= pkg.version %>/main.min.js',
+    },
+    test: {
+      entry: 'test/test-main.js',
```

Comment:
Should we rename this to main.js as well? Would work in my opinion as it is in the test folder.